platform-core-module/src/main/java/com/softicar/platform/core/module/CoreI18n.java

```diff
@@ -200,6 +200,14 @@ public interface CoreI18n extends EmfI18n {
 		.de("Tage");
 	I18n0 DAYS_SINCE_LAST_LOGIN = new I18n0("Days Since Last Login")//
 		.de("Tage seit letzten Einloggen");
+	I18n0 DBMS_CONFIGURATION = new I18n0("Dbms Configuration")//
+		.de("Dbms-Konfiguration");
+	I18n0 DBMS_CONFIGURATION_LOG = new I18n0("Dbms Configuration Log")//
+		.de("Dbms-Konfigurations-Log");
+	I18n0 DBMS_CONFIGURATION_LOGS = new I18n0("Dbms Configuration Logs")//
+		.de("Dbms-Konfigurations-Logs");
+	I18n0 DBMS_CONFIGURATIONS = new I18n0("Dbms Configurations")//
+		.de("Dbms-Konfigurationen");
```

Comment:
in natural languages, abbreviations are usually all uppercase, AFAIK
```suggestion
	I18n0 DBMS_CONFIGURATION = new I18n0("DBMS Configuration")//
		.de("DBMS-Konfiguration");
	I18n0 DBMS_CONFIGURATION_LOG = new I18n0("DBMS Configuration Log")//
		.de("DBMS-Konfigurations-Log");
	I18n0 DBMS_CONFIGURATION_LOGS = new I18n0("DBMS Configuration Logs")//
		.de("DBMS-Konfigurations-Logs");
	I18n0 DBMS_CONFIGURATIONS = new I18n0("DBMS Configurations")//
		.de("DBMS-Konfigurationen");
```